contracts/SecurityTokenRegistry.sol

```diff
@@ -234,9 +253,7 @@ contract SecurityTokenRegistry is EternalStorage, Proxy {
         require(_owner != address(0), "Owner should not be 0x");
         require(bytes(_ticker).length > 0 && bytes(_ticker).length <= 10, "Ticker length range (0,10]");
         // Attempt to charge the reg fee if it is > 0 POLY
-        uint256 tickerFee = getUintValue(TICKERREGFEE);
```

Comment:
`TICKERREGFEE` is now fees in USD, I suppose? All comments still say that it is the fee in Poly. Can you please update the comments if it's USD indeed.

Reply:
Same goes for `STLAUNCHFEE` / `_stLaunchFee`

Reply:
Thanks - I went through the code and updated all the comments I could find.